migrations/2020_01_14_130500_change_permission_groups_add_is_hidden.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::addColumns('groups', [
+    'is_hidden' => ['boolean', 'default' => 0]
```

Comment:
Can we switch all of this to `is_visible`, please? I'd like to avoid having to deal with double negations everywhere. :pray:

Reply:
This requires a follow up issue for Tags then? I followed that example 🙈

Reply:
Uh, right. Nah, let's rather be consistent then and leave it like it is.

We could discuss what we want in the frontend, though. A "hide" checkbox or one labeled "show"?

Reply:
I think the logic tags used is really clear. I think this is the right approach. Not sure how others perceive it.

PS the admin js still has to be verified as I ran into some issues..